### PR TITLE
Added leetcode problem 100

### PR DESCRIPTION
**Pull request by chandansahuji**
#### Checklist

- [x] Added description of change
- [x] Added file name matches [File name guidelines](https://github.com/TheAlgorithms/C/blob/master/CONTRIBUTING.md#File-Name-guidelines)
- [x] Search previous suggestions before making a new one, as yours may be a duplicate.
- [x] I acknowledge that all my contributions will be made under the project's license.

Notes:

### CODE REVIEW

**Comment by hussainiftikhar5242** [comment]
please assign this task to me @chandansahuji . i can do it.

**Comment by chandansahuji** [comment]
Hi @hussainiftikhar5242 ,its a PR(Pull Request),you can ask in the issues section ,to work on a particular issue.

File: leetcode/src/100.c [hunk 1]
```diff
@@ -0,0 +1,59 @@
+#include <stdio.h>
```
Comment (Panquesito7) [inline leetcode/src/100.c:1]: Could you please add a small description of the problem?

**Review by Panquesito7 — CHANGES_REQUESTED** [review]
Please resolve the merge conflicts (for the `leetcode/README.md` file) [**here**]8https://github.com/TheAlgorithms/C/pull/1026/conflicts).
Let us know if you need any help here or on our [**Discord server**](https://the-algorithms.com/discord)! 🙂

**Pull request closed without merging**